src/display/core/Settings.h

```diff
@@ -91,6 +92,7 @@ class Settings {
     void setHomeAssistantPassword(const String &homeAssistantPassword);
     void setHomeAssistantIP(const String &homeAssistantIP);
     void setHomeAssistantPort(int homeAssistantPort);
+    void setHomeAssistantTopic(int homeAssistantTopic);
```

Comment:
this is not implemented, you also need to load and save the value from storage

Reply:
added 7fb1c49